scss/civicrm/contact/pages/_tag-contacts.scss

```diff
@@ -245,3 +245,15 @@
     }
   }
 }
+
+#popupContainer {
```

Comment:
isn't `#popupContainer` generic? Why should it be under `_tag-contacts.scss` instead of being under a partial in `common/`? any style you write here will be applied to any popup, won't it?